lib/turn.rb

```diff
@@ -40,6 +40,7 @@ def find_lowest_cell_in_column(move, valid_columns)
     @board.board_grid.reverse.each do |row|
       if row[(valid_columns.index(move))].state == "."
         set_cell(@board.board_grid.index(row), valid_columns.index(move))
+        check_horizontal_win(@board.board_grid.index(row), valid_columns.index(move))
```

Comment:
ok now that I see this I totally understand your reasoning for the refactor we did last night. It does not make any sense to check for winning conditions in the find_lowest_cell method. Good call on the refactor. Gonna be a brain twister for me to built out the vertical method now that my brain in split in two but I'll try my best.